rise-financial-es6.js

```diff
@@ -214,6 +214,18 @@
       financial.params = params;
     }
 
+    _handleNoNetwork() {
+      this.$.data.getItem( this._getDataCacheKey(), ( cachedData ) => {
+        if ( cachedData ) {
+          this.fire( "rise-financial-response", cachedData );
+        } else {
+          this.fire( "rise-financial-no-network" );
```

Comment:
@donnapep Agreed, `rise-financial-no-network` sounds good. I'll change it, thanks